server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/bucket/DateHistogramIT.java

```diff
@@ -1338,18 +1336,7 @@ public void testIssue8209() throws InterruptedException, ExecutionException {
         internalCluster().wipeIndices("test8209");
     }
 
-    /**
-     * see issue #9634, negative dateHistogramInterval in date_histogram should raise exception
-     */
-    public void testExceptionOnNegativeInterval() {
-        try {
-            client().prepareSearch("idx")
-                    .addAggregation(dateHistogram("histo").field("date").interval(-TimeUnit.DAYS.toMillis(1)).minDocCount(0)).get();
-            fail();
-        } catch (IllegalArgumentException e) {
-            assertThat(e.toString(), containsString("[interval] must be 1 or greater for aggregation [date_histogram]"));
-        }
-    }
+    // TODO: add some tests for negative fixed and calendar intervals
```

Comment:
Will those tests be part of this PR or you plan to add them in another PR?

Reply:
Yeah, I was going to add them in later.  I don't think it's terribly urgent, and the test I removed only checked it on the legacy interval, so our coverage isn't worse for not adding them now.  And it's much less likely to be a problem because fixed & calendar intervals don't just accept bare longs, so a negative will almost definitely fail to parse.  Truthfully, I'm not even 100% sure it's worth doing, but I figured a small note was low friction.